erpnext/accounts/report/financial_statements.py

```diff
@@ -561,13 +561,12 @@ def get_accounts(company, root_type):
 		where company=%s and root_type=%s order by lft""", (company, root_type), as_dict=True)
 
 
-def filter_accounts(accounts, root_type, depth=10):
+def filter_accounts(accounts, depth=10):
```

Comment:
You need to remove `root_type` from `get_data` too.

Reply:
No, the root_type variable is used in the get_data function. In fact, it form a part of the core filter that is used in the get_accounts function called by the get_data function.